rs_workflows/staging.py

```diff
@@ -71,6 +71,7 @@ def update_stac_catalog(  # pylint: disable=too-many-locals
     collection_name: str,
     stac_file_info: dict,
     obs: str,
+    **kwargs,
```

Comment:
Why use a `**kwargs` and not just a regular parameter like this ? 
```python

def update_stac_catalog(  # pylint: disable=too-many-locals
    stac_client: StacClient,  # NOTE: maybe use RsClientSerialization instead
    collection_name: str,
    stac_file_info: dict,
    obs: str,
    update_assets,
):
```

Reply:
Can't you use an optional parameter ?
```python
def update_stac_catalog(  # pylint: disable=too-many-locals
    stac_client: StacClient,  # NOTE: maybe use RsClientSerialization instead
    collection_name: str,
    stac_file_info: dict,
    obs: str,
    update_assets: bool = False,
):
```

Reply:
> The point of using keyword arguments is to extend the functionality without affecting the one presented in 0.1.
I don't think any user is calling directly this `update_stac_catalog` function from v0.1 ? It's only called by our workflows ?

Reply:
I really don't understand why we spend so much time discussing this minor change.

I've told you already that the goal is not to change the function signature, which adding a extra parameter, even if is optional, does.